src/api/app/models/staging/staging_project.rb

```diff
@@ -8,6 +8,8 @@ class StagingProject < Project
     after_save :update_staging_workflow_on_backend
     after_destroy :update_staging_workflow_on_backend
     before_create :add_managers_group
+    after_create :create_project_log_entry
+    before_update :create_project_log_entry, if: proc { |staging_project| staging_project.staging_workflow_id_was.nil? }
```

Comment:
You probably could drop the `.nil?` by using `unless`. It might even be possible to shorten this to `unless: :staging_workflow_id_was`

Reply:
It's longer for sure, but it is also much clearer, so I prefer to leave it as it is.

Reply:
IMO, as soon as someone knows that `if` / `unless` accepts symbols it's not. And then you have much more stuff to mentally parse.